TRX/Tron-overview.md

```diff
@@ -126,7 +126,7 @@ Total Frozen TRX
 ```
 
 ## 5.3 Bandwidth consumption rules
-When there are available `Bandwidth`, no TRX is charged. If a transaction fee is charged, it will be recorded in the fee field in the transaction results. If no transaction fee is charged, meaning that corresponding bandwidth points have been deducted, the fee field will read “0”. There will only be a service charge after a transaction has been written into the blockchain. For more information on the fee field, please see also Transaction.Result.fee, with the corresponding proto file at https://github.com/TRONprotocol/protocol/blob/master/core/TRON.proto.
+When there are available `Bandwidth`, no TRX is charged. If a transaction fee is charged, it will be recorded in the fee field in the transaction results. If no transaction fee is charged, meaning that corresponding bandwidth points have been deducted, the fee field will read “0”. There will only be a service charge after a transaction has been written into the blockchain. For more information on the fee field, please see also `Transaction.Result.fee`, with the corresponding proto file at https://github.com/tronprotocol/protocol/blob/master/core/Tron.proto.
```

Comment:
"When there are available `Bandwidth`, no TRX is charged." should probably be "When there **is** `Bandwidth` available no TRX is charged."

Reply:
nice catch. I'll fix it.